web3.js/package.json

```diff
@@ -48,7 +48,8 @@
     "type:gen": "./scripts/typegen.sh",
     "lint": "set -ex; npm run pretty; eslint . --ext .js,.ts",
     "lint:fix": "npm run pretty:fix && eslint . --fix --ext .js,.ts",
-    "ok": "run-s lint test doc",
+    "type-check": "tsc -p tsconfig.json --noEmit --noUnusedParameters --noUnusedLocals",
```

Comment:
Can I make two requests?

1. Move the `noUnusedParameters` and `noUnusedLocals` config into `tsconfig.json`. That way, crucially, they'll show up as errors _in your editor's ‘problems’ pane_ so you can fix them right away, rather than learning about them much later at the CI step.
2. Rename the command in `package.json` to `type:check` for consistency.

Reply:
(Ordinarily I'd make these tiny changes myself, but I don't have edit permissions)

Reply:
Sure, in a minute

Reply:
@jordansexton I made the required updates, would you mind merging new changes?